package.json

```diff
@@ -29,15 +28,14 @@
     "kang": "0.0.8",
     "less": "1.7.5",
     "mime": "1.2.11",
-    "moray": "git+ssh://git@github.com:joyent/node-moray.git#b84ef0e",
+    "moray": "git+ssh://git@github.com:joyent/node-moray.git#31c0902",
     "node-uuid": "1.4.3",
     "promise": "6.1.0",
     "react": "0.13.0",
     "reactify": "1.0.0",
     "restify": "2.8.5",
```

Comment:
Trent Mick
Oct 19 11:35 PM
4.1.1
From a quick look at sdc-adminui.git against the restify compat changes described at https://github.com/joyent/rfd/tree/master/rfd/0059#restify  I think you are fine just updating from 2.8.5 to 4.1.1.